src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,17 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    int weight;
+    String model;
```

Comment:
access modifiers should be private, just add getters and setters so it will be possible to get access to these fields